src/pokemon.c

```diff
@@ -931,7 +931,7 @@ const u16 gSpeciesToNationalPokedexNum[] = // Assigns all species to the Nationa
     SPECIES_TO_NATIONAL(CHIMECHO),
 };
 
-const u16 gHoennToNationalOrder[] = // Assigns Hoenn Dex Pok�mon (Using National Dex Index)
+const u16 gHoennToNationalOrder[] = // Assigns Hoenn Dex Pok�mon (Using National Dex Index)
```

Comment:
Two of the `é` characters are messed up.

Reply:
idk how that happened, will try to figure out how to fix this